### PR TITLE
Add stdlib tests

### PR DESCRIPTION
**Pull request by jcivlin**
Testing **-L** (compile with stdlib) and **-p** (compile with package) options.

Notice that 
option **-p abs-path-to-stdlib** is equal to simple option **-L**.

### CODE REVIEW

File: language/tools/move-mv-llvm-compiler/tests/stdlib-tests.rs [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+//! Running the tests:
+//!
+//! ```
+//! cargo test -p move-mv-llvm-compiler --test stdlib-tests
+//! ```
+//!
+//! Running a specific test:
+//!
+//! ```
+//! cargo test -p move-mv-llvm-compiler --test stdlib-tests -- hash_tests.move
```
Comment (ksolana) [inline language/tools/move-mv-llvm-compiler/tests/stdlib-tests.rs:24]: Thanks for documenting these!

**Comment by ksolana** [comment]
nit: is it possible to change `-L` to something else. for gcc, clang `-L` is used for providing path to binaries. I think we should only keep a long form for this flag but it will be good to hear everyone's take.

**Comment by jcivlin** [comment]
> nit: is it possible to change `-L` to something else. for gcc, clang `-L` is used for providing path to binaries. I think we should only keep a long form for this flag but it will be good to hear everyone's take.

Changing is easy, but we better do it in the dedicated for this PR.

**Review by dmakarov — APPROVED** [review]

**Comment by jcivlin** [comment]
Thx for quick reviews!

**Comment by ksolana** [comment]
> > nit: is it possible to change `-L` to something else. for gcc, clang `-L` is used for providing path to binaries. I think we should only keep a long form for this flag but it will be good to hear everyone's take.
> 
> Changing is easy, but we better do it in the dedicated for this PR.

created https://github.com/solana-labs/move/pull/271

**Pull request merged**